modules/orca/main.tf

```diff
@@ -220,11 +237,11 @@ module "orca_ecs" {
   tags = var.tags
 }
 
-## orca_graph_ql - graphql module that sets up centralized db code
+## orca_graphql_1 - graphql module that sets up centralized db code
```

Comment:
I would note this the actual graphql deploymnet and that it needed the IAM roles and secretmanager available before it could be deployed. Maybe change the name to graphql_deploy instead of graphql_1? Thoughts?

Reply:
Added some comments. I am keeping the names deliberately generic to reduce Terraform redeploy messes.